examples/src/index.js

```diff
@@ -6,7 +6,6 @@ export { default as ContentExplorerMultiSelectModalContainerExamples } from './C
 export { default as ContextMenuWithSubmenuWithBoundariesElementExample } from './ContextMenuWithSubmenuWithBoundariesElementExample';
 export { default as DraftJSMentionSelectorExamples } from './DraftJSMentionSelectorExamples';
 export { default as MentionSelectorContainer } from './MentionSelectorContainer';
-export { default as DraggableListExamples } from './DraggableListExamples';
```

Comment:
I think we should leave these examples for now, since we havent deprecated styleguidist yet